packages/components/date-picker/src/utils.ts

```diff
@@ -31,3 +31,32 @@ export const isValidRange = (range: DayRange): boolean => {
 
   return isDayjs(left) && isDayjs(right) && left.isSameOrBefore(right)
 }
+
+type getDefaultValueParams = {
```

Comment:
As a type, I think our first letter should be capitalized.

Reply:
nice catch